apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.ts

```diff
@@ -41,19 +41,20 @@ router.post(
 router.get(
   '/variant/:variant_id(\\d+)/submission/:submission_id(\\d+)',
   asyncHandler(async (req, res) => {
-    const { submissionPanel, extraHeadersHtml } = await renderPanelsForSubmission({
+    const panels = await renderPanelsForSubmission({
       submission_id: req.params.submission_id,
       question_id: res.locals.question.id,
       instance_question_id: null,
       variant_id: req.params.variant_id,
       user_id: res.locals.user.user_id,
       urlPrefix: res.locals.urlPrefix,
       questionContext: 'instructor',
-      csrfToken: null,
+      // This is only used by score panels, which are not rendered in this context.
       authorizedEdit: null,
```

Comment:
In the public page you changed this to false, any reason why this one is still null?

Reply:
Just hadn't taken a consistency pass just yet! I think `false` makes the most sense to use everywhere, do you agree?

Reply:
Changed in fa56ada.